### DISCUSSION TITLE
展示文件时文件过多会导致卡顿

### DISCUSSION THREAD

**Discussion by allyxmiko** [1Panel-dev/1Panel#5254] [Show and tell]
### 联系方式

1713069822@qq.com

### 1Panel 版本

v1.10.9-lts

### 问题描述

主机 -> 文件下，文件过多滚动时会导致动画卡顿，延迟。
CPU：i7-11800H
内存：32G 3200Mhz
浏览器： Edge 版本125.0.2535.79

### 重现步骤

https://github.com/1Panel-dev/1Panel/assets/53589496/be079d93-3b9c-4b6a-9658-9c948f30d083

如视频所示，不滚动时CPU占用很低，滚动滚轮后CPU占用上升，滚动卡顿，不跟鼠标（示例目录为/etc）

### 期待的正确结果

滚动流畅，跟鼠标

**Comment by wanghe-fit2cloud** [comment]
感谢反馈！上述现象是由于 1Panel 使用的第三方 SDK 读取速度较慢，加上文件列表内容过多导致的。我们将在后续版本中尝试优化文件列表查询接口。
